go/kvstore/boxer.go

```diff
@@ -31,74 +31,26 @@ func NewKVStoreBoxer(g *libkb.GlobalContext) *KVStoreRealBoxer {
 	}
 }
 
-type signatureElements struct {
-	EntryID           keybase1.KVEntryID
-	ClearBytes        []byte
-	Revision          int
-	Nonce             [24]byte
```

Comment:
i removed the nonce from the associated data object. i think that's ok to do, but tell me if you think otherwise.

Reply:
it's ok. I would like to see some testing around the nonce though. For example:
- generate two nonces and assert they are different.
- encrypt something and assert the nonce field was actually set to something non zero
- encrypt something with a good nonce and try to decrypt with a different nonce.

Reply:
i can add a couple nonce-related tests.

Reply:
done.